omf-python/src/element.rs

```diff
@@ -30,6 +31,14 @@ impl PyElement {
         self.0.description.clone()
     }
 
+    #[getter]
+    /// Element metadata.
+    fn metadata(&self) -> PyResult<String> {
+        let metadata = serde_json::to_string(&self.0.metadata)
+            .map_err(|e| OmfException::py_err(DeserializationFailed(e)))?;
```

Comment:
Wrong error code. This is a serialisation operation, not deserialisation.

It would also be better to construct and return a dict containing None, int, float, str, list, and dict objects rather than forcing all callers to do their own JSON deserialisation.

Reply:
Yep, I was going to look into that.

Reply:
It's now converted into a python dict.